frontend-react/src/pages/Upload.tsx

```diff
@@ -40,7 +42,7 @@ export const Upload = () => {
         `Please resolve the errors below and upload your edited file. Your file has not been accepted.`
     );
 
-    const client = senderClient(authState);
+    const client = `${getStoredOrg()}.${getStoredSenderName()}`;
```

Comment:
Is there a situation where the user has two tabs open and logs out in the 2nd tab and then `getStoredOrg()` and/or `getStoredSenderName()` fail (which currently return undefined).

I guess client variable will then become `undefined.undefined` and doesn't throw?

Reply:
Hmm, this sounds like it's a larger issue overall since logout always calls `sessionStorage.clear()`; all API calls would have `Bearer undefined` and `Organization: undefined` in the headers. With a SPA, what's the most common way of managing a session storage across two tabs with the same app open?